src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialog.java

```diff
@@ -33,21 +34,21 @@ public AboutDialog(ProductInfo info)
       super(Roles.getDialogRole());
       RStudioGinjector.INSTANCE.injectMembers(this);
 
-      setText("About " + editionInfo_.editionName());
-      ThemedButton OKButton = new ThemedButton("OK", (ClickEvent) -> closeDialog());
+      setText(messages_.title(editionInfo_.editionName()));
+      ThemedButton OKButton = new ThemedButton(constants_.okBtn(), (ClickEvent) -> closeDialog());
       addOkButton(OKButton);
 
       if (editionInfo_.proLicense() && Desktop.hasDesktopFrame())
       {
-         ThemedButton licenseButton = new ThemedButton("Manage License...", (ClickEvent) ->  {
+         ThemedButton licenseButton = new ThemedButton(constants_.manageLicenseBtn(), (ClickEvent) ->  {
             closeDialog();
             editionInfo_.showLicense();
          });
          addLeftButton(licenseButton, ElementIds.ABOUT_MANAGE_LICENSE_BUTTON);
       }
       contents_ = new AboutDialogContents(info, editionInfo_);
       setARIADescribedBy(contents_.getDescriptionElement());
-      setWidth("600px");
+      setWidth("600px"); //$NON-NLS-1$
```

Comment:
Yes, `$NON-NLS-1$` is used to suppress warnings for hardcoded string literals.  I don't know if it originates from Eclipse or if it comes from elsewhere, but it is recognized by at least Eclipse and IntelliJ.

That is something I wanted to bring up actually.  Do you have a preference how (and if) to supress these warnings on valid string literals (like the `600px` above)?  The two ways I've seen so far are:
* inline `$NON-NLS-1$` comment, which suppresses the warning for this line only
* java annotations, which can suppress the warning more broadly (for example, we can suppress the warning for any string literal passed to the `width` parameter of `setWidth`).  These add some extra XML files to the project, but save a lot of code changes for examples like `setWidth`.  I know IntelliJ respects the annotations, but I don't know if Eclipse does

This will come up a lot more in some of the future PRs.  I recently went through the ~16000 string literals and suppressed warnings for any that looked like they should not be localized (~60% of them), either by `$NON-NLS-1$` or annotation.  If the annotations are acceptable they're much less work for many cases.

Reply:
Let's table that discussion for a bit (don't know enough to have an opinion on which approach would be preferable and need to do some playing with it).

Reply:
Agreed.  I can give a better example of how it looks later too